### DISCUSSION TITLE
Search results - query template crawler through content of site pages

### DISCUSSION THREAD

**Discussion by LolzerSVK** [microsoft-search/pnp-modern-search#3791] [Q&A] [answered]
Hi, is it possible to create a query template in the Search Results webpart that will crawl through the content of site pages (aspx links)?
Because I have aspx links (of site pages) stored inside a SharePoint list and there are metadata inside this list regarding the aspx page.

Thanks for any info, whether its possible to crawl through the page content or not.

**Comment by wobba** [comment, marked as answer]
No. Pages are crawled by themselves (if in SharePoint), and list items by themselves.
